server/appenders/cassandra/appender/src/main/java/org/kaaproject/kaa/server/appenders/cassandra/appender/CassandraLogEventDao.java

```diff
@@ -0,0 +1,212 @@
+import com.datastax.driver.core.querybuilder.Batch;
+import com.datastax.driver.core.querybuilder.Insert;
+import com.datastax.driver.core.querybuilder.QueryBuilder;
+import com.google.common.util.concurrent.ListenableFuture;
+import org.kaaproject.kaa.common.dto.logs.LogEventDto;
+import org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraBatchType;
+import org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraCompression;
+import org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraConfig;
+import org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraCredential;
+import org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraServer;
+import org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraSocketOption;
+import org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraWriteConsistencyLevel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.concurrent.Future;
+
+public class CassandraLogEventDao implements LogEventDao {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CassandraLogEventDao.class);
+
+    private static final String ID_COLUMN = "id";
+    private static final String HEADER_COLUMN = "header";
+    private static final String EVENT_COLUMN = "event";
+
+    private static final String TABLE_NAME = "$table_name";
+    private static final String KEYSPACE_NAME = "$keyspace_name";
+    private static final String CREATE_TABLE = "CREATE TABLE IF NOT EXISTS $keyspace_name.$table_name (" + ID_COLUMN + " text, "
+            + HEADER_COLUMN + " text, " + EVENT_COLUMN + " text, PRIMARY KEY (id) );";
+
+    private final Random RANDOM = new Random();
+
+    private Cluster cluster;
+    private Session session;
+    private CassandraBatchType batchType;
+    private ConsistencyLevel writeConsistencyLevel;
+    private String keyspaceName;
+
+    public CassandraLogEventDao(CassandraConfig configuration) throws UnknownHostException {
+        LOG.info("Init cassandra log event dao...");
+        if (configuration != null) {
+            keyspaceName = configuration.getKeySpace();
+            List<InetSocketAddress> clusterNodes = new ArrayList<>();
+            List<CassandraServer> nodes = configuration.getCassandraServers();
+            for (CassandraServer node : nodes) {
+                clusterNodes.add(new InetSocketAddress(InetAddress.getByName(node.getHost()), node.getPort()));
+            }
+
+            Cluster.Builder builder = Cluster.builder().addContactPointsWithPorts(clusterNodes);
+            if (LOG.isTraceEnabled()) {
+                LOG.trace("Init cassandra cluster with nodes {}", Arrays.toString(clusterNodes.toArray()));
```

Comment:
This may have higher log level. Even info is ok, if we don't have any secure info there